xknx/devices/light.py

```diff
@@ -486,11 +491,12 @@ async def set_color(
         """
         if white is not None:
             if self.supports_rgbw:
-                if self.rgbw.initialized:
+                if self.rgbw.has_any_group_address:
```

Comment:
this is out of scope for this PR but we should consider to use `writable` property when trying to write and have some exception if we don't have one.